intel_owl/settings.py

```diff
@@ -145,7 +146,7 @@
 # these two are needed to enable priority and correct tasks execution
 CELERY_TASK_ACKS_LATE = True
 CELERY_WORKER_PREFETCH_MULTIPLIER = 1
-CELERY_TASK_DEFAULT_QUEUE = "analyzers_queue"
+CELERY_QUEUES = os.environ.get("CELERY_QUEUES", "default").split(".")
```

Comment:
I think `CELERY_QUEUES` variable should be defined in the `env_file_app`, right ?

Reply:
I get your point. The problem is that we should then provide two different `env_file_app`, one for the `multi-queue`, one for the standard. Since I personally don't like to provide to users too many different files, I decided to use the `environment` keyword inside the docker-compose-multi-queue.yaml to describe this behaviour.

Reply:
Well, thinking about it a little bit more, you are probably right. Splitting the environment variables is not a good practice.
Maybe I can add something like this inside the `env_file_app`, what do you think?
```
...
CELERY_QUEUES="default"
# USE THIS FOR THE MULTI-QUEUE
# CELERY_QUEUES="default.local.long"
...
```

Reply:
ok, my bad actually; I didn't notice that you put it inside the compose file itself, I thought it was missing. Then I think your intial implementation was correct because there's no point in leaving this upto the user to decide. 

Just one thing, instead of `default.local.long` can you change it to `default,local,long` and then split on the basis of `,` ?

Reply:
ok, my bad actually; I didn't notice that you put it inside the compose file itself, I thought it was missing. Then I think your intial implementation was correct because there's no point in leaving this upto the user to decide since it is tied with the compose file they choose.

Just one thing, instead of `default.local.long` can you change it to `default,local,long` and then split on the basis of `,` ?

Reply:
Yeah, sure